packages/commercetools/theme/components/AppHeader.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <SfHeader logo="/icons/logo.svg">
+    <template #navigation>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/l/cat">
+          WOMEN
+        </nuxt-link>
+      </SfHeaderNavigationItem>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/l/cat">
+          KIDS
+        </nuxt-link>
+      </SfHeaderNavigationItem>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/l/cat">
```

Comment:
How about wrapping `SfHeaderNavigation` with `nuxt-link`? Seems like a better idea from users PoV - the whole tile will be clickable

Reply:
Hi @filrak 
It didn't change much on desktop, maybe had effect on mobile.
Do you think we can also wrap the logo in a link to homepage? Right now its not clickable and its kind of akward

Reply:
Ok. If the styling is for text, not outer wrapper then it indeed doesn't make sense. Thanks ;)
Regarding logo - it definitely should be clickable!